content/applications/websites/ecommerce/managing_products/price_management.rst

```diff
@@ -0,0 +1,204 @@
+- :guilabel:`Website`: The website(s) on which the program is available. Leave the field to make it
+  available on *all* websites;
+- :guilabel:`Point of Sale`: The :abbr:`PoS (Point of Sale)` in which the program is available.
+  Leave empty to make it available on *all* :abbr:`PoS (Point of Sale)`.
+
+.. image:: price_management/price-programs.png
+   :align: center
+   :alt: Program options
+
+You can see the number of promotional items available per program in the upper-right corner, under
+the corresponding tab.
+
+.. image:: price_management/price-programs-items.png
+   :align: center
+   :alt: Program items tab
+
+Conditional rules
+~~~~~~~~~~~~~~~~~
+
+Under the :guilabel:`Rules & Rewards` tab, you can set the conditions and points value depending on
+the program type you selected. Click :guilabel:`Add` in the :guilabel:`Conditional rules` column to
+add **conditions**.
+
+- :guilabel:`Minimum Quantity`: The minimum quantity that must be bought to access the reward;
+- :guilabel:`Minimum Purchase`: The minimum amount (in currency) :guilabel:`tax Included` or
+  :guilabel:`tax Excluded` to access the reward. You can combine both quantity *and* amount;
+- :guilabel:`Products`: The product(s) to which the program applies;
+- :guilabel:`Categories`: The category of products to which the program applies. Select
+  :guilabel:`All` to apply to *all* categories;
+- :guilabel:`Product Tag:` Select a tag to apply the program to products containing that tag;
+- :guilabel:`Grant`: Set the number of points the customer can earn :guilabel:`per order`,
+  :guilabel:`per currency spent`, or :guilabel:`per unit paid`.
+
+.. image:: price_management/price-conditions.png
+   :align: center
+   :alt: Conditional rules to get rewards through a program type
+
+Rewards
+~~~~~~~
+
+Click :guilabel:`Add` in the :guilabel:`Rewards` column to add **rewards**. According to the type of
+reward selected, different options may be available.
+
+- :guilabel:`Reward Type`: Select the reward type among :guilabel:`Discount`,
+  :guilabel:`Free product`, and :guilabel:`Free Shipping`;
+- :guilabel:`Discount`: The discounted amount in :guilabel:`%`, :guilabel:`currency per point`, or
+  :guilabel:`currency per order` on either the :guilabel:`Order`, the :guilabel:`Cheapest Product`,
+  or the :guilabel:`Specific Products`;
+- :guilabel:`In exchange of`: Enter the amount of tokens or points earned by the customer to be
+  exchanged for the reward;
+- :guilabel:`Description on order`: The description of the reward displayed upon checkout;
+- :guilabel:`Max Discount`: The maximum amount (in currency) granted by the discount. Leave to `0`
+  for unlimited amount;
+- :guilabel:`Discounted Products`: The selected product(s) eligible for discount;
+- :guilabel:`Discounted Prod. Categories`: The product category eligible for discount. Select
+  :guilabel:`All` for all categories;
+- :guilabel:`Discounted Prod. Tag`: The products eligible for discount according to their tag.
```

Comment:
```suggestion
Rewards
~~~~~~~

Click :guilabel:`Add` in the :guilabel:`Rewards` column to add **rewards**. According to the type of
reward selected, different options may be available.

- :guilabel:`Reward Type`: Select the reward type among :guilabel:`Discount`,
  :guilabel:`Free product`, and :guilabel:`Free Shipping`;
- :guilabel:`Discount`: The discounted amount in :guilabel:`%`, :guilabel:`currency per point`, or
  :guilabel:`currency per order` on either the :guilabel:`Order`, the :guilabel:`Cheapest Product`,
  or the :guilabel:`Specific Products`;
- :guilabel:`In exchange of`: Enter the amount of tokens or points earned by the customer to be
exchanged for the reward;
- :guilabel:`Description on order`: The description of the reward displayed upon checkout;
- :guilabel:`Max Discount`: The maximum amount (in currency) granted by the discount. Leave as `0`
  for unlimited amounts;
- :guilabel:`Discounted Products`: The selected product(s) eligible for discount;
- :guilabel:`Discounted Prod. Categories`: The product category eligible for discount. Select
  :guilabel:`All` for all categories;
- :guilabel:`Discounted Prod. Tag`: The products eligible for discount according to their tag.
```

Reply:
Once more, not sure about that blank space. I changed your suggestion "as" to "on"